### PR TITLE
Add scrypted enhanced app

### PR DESCRIPTION
**Pull request by 3nigma71**

### CODE REVIEW

**Comment by keriati** [comment]
**Running Tests**

Testing Scrypted
Failed: should have an icon file
Icon is too small: { width: 0, height: 0 }

Testing Scrypted
Failed: should have an Icon with min width 100px and min height 100px when PNG

Some Tests Failed!
Error: Process completed with exit code 1.

**Comment by Rygle** [comment]
scrypted
Scrypted 256x256 tinypng

File: Scrypted/Scrypted.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public function test()
+    {
+        $test = parent::appTest($this->url('status'));
+        echo $test->status;
+    }
+
+    public function livestats()
+    {
+        $status = 'inactive';
+        $res = parent::execute($this->url('status'));
```
Comment (mvdkleijn) [inline Scrypted/Scrypted.php:23]: Is this the actual endpoint? Looks like the actual retrieval of livestats isn't implemented.

File: Scrypted/app.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "f18e7d860a9886eb86bb406d2e9afdc6eb0bf67f",
+    "name": "Scrypted",
+    "website": "https://www.scrypted.app/",
+    "license": "W3C Software Notice and License (2002-12-31)",
+    "description": "Integration platform",
+    "enhanced": true,
+    "tile_background": "dark",
+    "icon": "scrypted.png"
```
Comment (mvdkleijn) [inline Scrypted/app.json:9]: Icon missing. It is required. SVG format preferred.

File: Scrypted/app.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "f18e7d860a9886eb86bb406d2e9afdc6eb0bf67f",
+    "name": "Scrypted",
+    "website": "https://www.scrypted.app/",
+    "license": "W3C Software Notice and License (2002-12-31)",
+    "description": "Integration platform",
```
Comment (mvdkleijn) [inline Scrypted/app.json:6]: That description is a little too small IMHO.

File: Scrypted/livestats.blade.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<ul class="livestats">
```
Comment (mvdkleijn) [inline Scrypted/livestats.blade.php:1]: I don't think scrypted has a queue size and speed? Again, this looks like a copy-paste and unimplemented.

**Review by mvdkleijn — CHANGES_REQUESTED** [review]

**Comment by mvdkleijn** [comment]
Closing this PR since it looks like an unimplemented copy of the dev templates for an enhanced app.

Also:
- no reaction from pr owner
- no icon
- no real description

**Pull request closed without merging**